### PR TITLE
add some missing fields using a mystery script

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: js/tidex.js [hunk 1]
```diff
@@ -528,6 +528,10 @@ module.exports = class tidex extends Exchange {
             'filled': filled,
             'fee': undefined,
             // 'trades': this.parseTrades (order['trades'], market),
+            'info': undefined,
```
Comment (kroitor) [inline js/tidex.js]: This is a breaking change considering the `extend` below. Need to switch the ordering in the `extend`...

Comment (kroitor) [reply js/tidex.js]: ... everywhere...

File: js/stex.js [hunk 2]
```diff
@@ -933,6 +933,7 @@ module.exports = class stex extends Exchange {
             'remaining': remaining,
             'status': status,
             'trades': trades,
+            'fee': undefined,
```
Comment (kroitor) [inline js/stex.js]: This line should not be here, if `fees` is preset.

File: js/tidex.js (hunk 1 above)
Comment (frosty00) [reply js/tidex.js]: imo there shouldn't be an extend there, just a regular assignment

Comment (kroitor) [reply js/tidex.js]: @frosty00 yep, you're right, an assignment would be better

**Comment by frosty00** [comment]
ignore the commit message

**Comment by kroitor** [comment]
https://travis-ci.org/github/ccxt/ccxt/builds/668195466#L2532
https://travis-ci.org/github/ccxt/ccxt/builds/668195466#L2558

**Pull request merged**